### DISCUSSION TITLE
fedora 39 packages

### DISCUSSION THREAD

**Discussion by hboetes** [puppetlabs/community#66] [General]
Looking at https://yum.puppetlabs.com/ the latest fedora supported release is 36, which has reached EOL since more than a year.  https://docs.fedoraproject.org/en-US/releases/eol/
The current release is 39 and the upcoming release is 40, for which the beta has already been released.

I tried running the packages provided by fedora, but then there's always some other error.

Could you be so kind as to bring the package generator up to date? Thanks!

**Comment by mhashizume** [comment]
Closing as a dupe of https://github.com/puppetlabs/puppet/issues/9320
